Update recommendations in the documents (#241)
scalar-labs/helm-charts: docs/configure-custom-values-scalardl-ledger.md

### Image configurations

> You must set `ledger.image.repository`. Please specify the container repository information that you pull the ScalarDL Ledger container image.

Author: As I mentioned in another comment, ScalarDL Ledger is a private (commercial license) product.
So, your assumption is correct. This configuration is required.
I will apply your suggestion.
Thank you!

  scalarLedgerConfiguration:
    ledgerLogLevel: INFO
```

> ### Taints/Tolerations configurations (Optional based on your environment)

Reviewer: ```suggestion
### Taint and toleration configurations (Optional based on your environment)
```

> If you want to control pod deployment using the taints and tolerations of Kubernetes, you can use `ledger.tolerations`.
> 
> You can configure them using the same syntax as the tolerations of Kubernetes. So, please refer to the official document [Taints and Tolerations](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/) for more details on the tolerations configuration of Kubernetes.

Reviewer: Some minor suggestions here. (Changes made here are consistent across other docs in this PR.)

```suggestion
If you want to control pod deployment by using the taints and tolerations in Kubernetes, you can use `ledger.tolerations`.

You can configure taints and tolerations by using the same syntax as the tolerations in Kubernetes. For details on configuring tolerations in Kubernetes, see the official Kubernetes documentation [Taints and Tolerations](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/).
```
